nixos/tests/cosmic.nix

```diff
@@ -42,6 +42,9 @@
       # infrastructure.
       jq
       lswt
+
+      firefox
```

Comment:
I am not convinced that we should test firefox and mpv here?
Especially since that might mean we need to build firefox on unstable potentially regularly.